source/guide_libreoffice.rst

```diff
@@ -0,0 +1,444 @@
+.. highlight:: console
+
+.. author:: jorsn <johannes@jorsn.eu>
+
+.. tag:: office-suite
+.. tag:: collaborative-editing
+.. tag:: spreadsheet
+.. tag:: presentation
+.. tag:: web
+.. tag:: docker
+.. tag:: udocker
+
+.. sidebar:: About
+
+  .. image:: _static/images/libreoffice.svg
+  .. image:: _static/images/collabora-code.svg
+      :align: center
+
+##################
+LibreOffice Online
+##################
+
+.. tag_list::
+
+`LibreOffice Online <LOOL_>`_ is a free online office suite based on the
+desktop office suite LibreOffice. It comprises a word processor, a spreadsheet
+and a presentation software. For usage it must be integrated with a file host
+which provides file access via WOPI or WebDAV, such as :lab:`Nextcloud
```

Comment:
Please add this information to a info box at the beginning of the guide. If I understand this right, in a production setup you'll need Nextcloud or Seafile to actually use this guide, right?

Reply:
Nextcloud, Seafile or any other WOPI or WebDAV host.